static/app/components/searchSyntax/renderer.tsx

```diff
@@ -19,30 +19,49 @@ type Props = {
    * highlight active tokens and trigger error tooltips.
    */
   cursorPosition?: number;
+  /**
+   * A function that returns a warning message for a given filter key
+   * will only show a render a warning if the value is truthy
+   */
+  getFilterWarning?: (key: string) => React.ReactNode | undefined;
```

Comment:
Would it make more sense to be provided the entire filter token?

Reply:
agree, thx! 👍